src/PointwiseFunctions/GeneralRelativity/CMakeLists.txt

```diff
@@ -6,10 +6,19 @@ set(LIBRARY GeneralRelativity)
 set(LIBRARY_SOURCES
     Christoffel.cpp
     ComputeGhQuantities.cpp
-    ComputeSpacetimeQuantities.cpp
+    DerivativesOfSpacetimeMetric.cpp
+    ExtrinsicCurvature.cpp
     IndexManipulation.cpp
+    InverseSpacetimeMetric.cpp
     KerrSchildCoords.cpp
+    Lapse.cpp
     Ricci.cpp
+    Shift.cpp
+    SpacetimeMetric.cpp
+    SpacetimeNormalOneForm.cpp
+    SpacetimeNormalVector.cpp
+    SpatialMetric.cpp
+    TimeDerivativeOfSpacetimeMetric.cpp
     WeylElectric.cpp
     )
 
```

Comment:
for my understanding, why are the headers here not declared like the `GeneralizedHarmonic` ones?

Reply:
They should be. We are in the process of switching over to declaring the headers (@fmahebert is doing a lot of the work here, which I am extremely appreciative of!!!). In the future we will need to keep track of all source and header files. It would be nice to add them already, but see if it conflicts with any modernization PRs of CMake that are open

Reply:
I have not yet touched any of the CMakeLists.txt in `PointwiseFunctions`, partly because I was waiting for this PR to get merged in first. For me it makes little difference to update one additional CMakeLists.txt file, so I would be happy for us to merge @geoffrey4444's PR as-is and I will modernize the CMake afterwards. But if @geoffrey4444 wants a try at updating a CMakeLists.txt, that's great too 🙂 

@moxcodes which GeneralizedHarmonic headers are you referring to?